File: C/369.c
#include <stdio.h>
#include <stdlib.h>

// 369 - Contando en la Arena - Iván - https://github.com/wildfireOfMine

int main()
{
    int number = 0;
    scanf("%d", &number);
    while (number != 0) {
        int i;
        for (i = 0; i<number; i++) {
            printf("1");
        }
        printf("\n");
        scanf("%d", &number);
    }
    return 0;
}

File: Java/369.java
import java.util.Scanner;

/**
 *
 * 369 - Contando en la Arena - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int numero = sc.nextInt();
        while (numero != 0) {
            for (int i = 1; i <= numero; i++) {
                System.out.print("1");
            }
            System.out.println();
            numero = sc.nextInt();
        }

    }

}
